### PR TITLE
Added basic Delta Orion Controller support

### PR DESCRIPTION
**Pull request by craig-nokia**
Please give a short description what your pull request is for

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: includes/definitions/deltaorion.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+os: deltaorion
+group: delta
+text: 'Delta Orion Controller'
+type: environment
+icon: delta
+discovery_modules:
+    ntp: false
```
Comment (Jellyfrog) [inline includes/definitions/deltaorion.yaml:7]: Please Remove all these and under poller too. If not needed due to bugs etc

File: tests/data/deltaorion_deltaorion.json [hunk 2]
```diff
@@ -0,0 +1,437 @@
...
+    "os": {
+        "discovery": {
+            "devices": [
+                {
+                    "sysName": "<private>",
+                    "sysObjectID": ".1.3.6.1.4.1.20246",
+                    "sysDescr": "Orion Power System Controller. Delta Electronics (Switzerland) AG.",
+                    "sysContact": "<private>",
+                    "version": "B8.35Build03",
+                    "hardware": null,
```
Comment (Jellyfrog) [inline tests/data/deltaorion_deltaorion.json:11]: Is there any Oid for hardware?

File: includes/definitions/deltaorion.yaml (hunk 1 above)
Comment (craig-nokia) [reply includes/definitions/deltaorion.yaml]: Removed.

File: tests/data/deltaorion_deltaorion.json (hunk 2 above)
Comment (craig-nokia) [reply tests/data/deltaorion_deltaorion.json:11]: There is no hardware discription when I walk the device.  Here is what is known...
dcSiteName.0 = XXX
dcSystemName.0 = XXX
dcSystemDateTime.0 = 2021-04-12T09:06:52
dcSoftwareVersion.0 = B8.35Build03
dcSystemInfo.5.0 = 1

**Review by Jellyfrog — APPROVED** [review]

**Pull request merged**